services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -916,6 +918,59 @@ async def get_project_comment(self, comment_id: CommentID) -> ProjectsCommentsDB
         async with self.engine.acquire() as conn:
             return await get_project_comment(conn, comment_id)
 
+    #
+    # Project Wallet
+    #
+
+    async def get_project_wallet(
```

Comment:
Is there a way to move this piece away from `projects.db` ... it iwll simplify refactoring of this module as we want

Reply:
yes, i will move it when we decide how we will proceed with the refactoring of `db.py`